content/doc/developer/forms/form-validation.adoc

```diff
@@ -45,6 +45,14 @@ public FormValidation doCheckThreads(@QueryParameter("value") final String meani
 
 This naming convention is how Jenkins wires up your form validation method. The return type also must be FormValidation, which offers various static factory methods to create an instance. Form validation can be of 3 different kinds: `ok`, `warning` or `error`. Validation message can either be represented in plaintext or use HTML markup. Several validations of different kinds can be reported wrapping then into one using aggregate factory method.
 
+=== To set up form validation within a Jelly template
+
+To configure form validation within a Jelly template to call doCheckXxx methods defined in a particular class in Jenkins, you'll indeed utilize the checkUrl and checkDependsOn attributes. Here's a breakdown of how these attributes are used:
```

Comment:
```suggestion
To configure form validation within a Jelly template to call `doCheckXxx` methods, defined in a particular class in Jenkins, you'll need to utilize the `checkUrl` and `checkDependsOn` attributes.
Here's a breakdown of how these attributes are used:
```
Suggestion to use code span for some of the items that seem to be something the user would enter.
Suggestion to have one full sentence per line and adjust some punctuation.